app/views/__init__.py

```diff
@@ -2,10 +2,12 @@
 from .auth import bp as auth_bp
 from .services_adm import bp as service_bp
 from .vehicles_adm import bp as vehicle_bp
+from .appointments_adm import bp as appointment_bg
```

Comment:
do you mean `appointment_bp`? just to make it consistent with other blueprints

Reply:
Nice catch! Thanks. The p key is so closed to the g key. That's the reason why Ha ha